pkg/commands/ls.go

```diff
@@ -113,7 +113,14 @@ func mergeStagingComponents(deployed []apis.ComponentMeta, env *types.EnvMeta, i
 					v.Traits[j] = t
 				}
 				aspec.Components[i] = v
+				/* Before any traits is attached to the workload, if ApplicationConfiguration is retrieved through
+				OpenAPI, the .spec.Components[].Traits is nil; if it's resembled by local file, .spec.Components[].Traits
```

Comment:
What do you mean by "resembled by local file"?

Reply:
We are comparing appConfig with the one from cached in local file to check whether the status of the application is staging. For details, please refer to https://github.com/oam-dev/kubevela/blob/master/pkg/appfile/service.go#L73.